src/dotnet/Fable.Compiler/CLI/Agent.fs

```diff
@@ -207,7 +207,8 @@ let startAgent () = MailboxProcessor<Command>.Start(fun agent ->
                 let msg = Parser.parse msg
                 // lazy sprintf "Received message %A" msg |> Log.logVerbose
                 let isUpdated, state, activeProject = updateState checker state msg
-                let com = Compiler(msg.path, activeProject, Parser.toCompilerOptions msg)
+                let fableCore = (Path.getRelativePath msg.path activeProject.FableCore).TrimEnd('/')
+                let com = Compiler(fableCore, msg.path, activeProject, Parser.toCompilerOptions msg)
```

Comment:
Hmm, Fable 2 now uses [the CLI to pass an alternative `fable-core` location](https://github.com/fable-compiler/Fable/blob/37bb279213b21a80a00fe125e5859984bafcbf0e/src/dotnet/Fable.Compiler/CLI/Main.fs#L112-L117). This is because it was annoying in Fable 1 to edit the webpack.config.js every time I wanted to test a project with Fable development version. Maybe it's better to use it for the benchmark [as in here](https://github.com/fable-compiler/Fable/blob/37bb279213b21a80a00fe125e5859984bafcbf0e/src/tools/quicktest.sh#L41) and deprecate the option in the JS client (I think nobody else uses it anyway)?

Reply:
- Perhaps, but OTOH it's a config option so it makes sense for it to be in the config object that is being passed to the splitter, and everywhere else. You can have an override in the CLI line as you already do, it's a bit strange with the mutable global and such, but whatever works.